ui/src/dashboards/actions/thunks.ts

```diff
@@ -229,6 +233,34 @@ export const getDashboards = () => async (
     )
 
     dispatch(setDashboards(RemoteDataState.Done, dashboards))
+
+    if (!dashboards.result.length) {
+      return
+    }
+
+    Object.values(dashboards.entities.dashboards)
+      .map(d => {
+        return {
+          id: d.id,
+          cells: d.cells.map(c => dashboards.entities.cells[c]),
+        }
+      })
+      .forEach(e => {
+        const viewsData = viewsFromCells(e.cells, e.id)
+
+        const normViews = normalize<View, ViewEntities, string[]>(
+          viewsData,
+          arrayOfViews
+        )
+
+        const normCells = normalize<Dashboard, DashboardEntities, string[]>(
+          e.cells,
+          arrayOfCells
+        )
+
+        dispatch(setViews(RemoteDataState.Done, normViews))
+        dispatch(setCells(e.id, RemoteDataState.Done, normCells))
+      })
```

Comment:
i really dislike single letter variable names - they make it very hard to find patterns and to track down loose ends in global searches and require a little extra brainpower to process while reading. writing out a variable name costs us very little.

```suggestion
      .map(dashboard => {
        return {
          id: dashboard.id,
          cells: dashboard.cells.map(cell => dashboards.entities.cells[cell]),
        }
      })
      .forEach(entity => {
        const viewsData = viewsFromCells(entity.cells, entity.id)

        const normViews = normalize<View, ViewEntities, string[]>(
          viewsData,
          arrayOfViews
        )

        const normCells = normalize<Dashboard, DashboardEntities, string[]>(
          entity.cells,
          arrayOfCells
        )

        dispatch(setViews(RemoteDataState.Done, normViews))
        dispatch(setCells(entity.id, RemoteDataState.Done, normCells))
      })
```